build.zig

```diff
@@ -427,7 +427,7 @@ pub fn build(b: *std.Build) !void {
     const optimization_modes = chosen_opt_modes_buf[0..chosen_mode_index];
 
     const fmt_include_paths = &.{ "lib", "src", "test", "tools", "build.zig", "build.zig.zon" };
-    const fmt_exclude_paths = &.{"test/cases"};
+    const fmt_exclude_paths = &.{ "test/cases", "test/behavior/zon" };
```

Comment:
Why are these excluded from formatting?

Reply:
Auto format is good on code for readability, but normalizing the zon test cases to standard formatting reduces coverage. Tests cover invalid syntax and non standard formatting

Reply:
Hmm... I think incorrect formatting is something we could better test with some kind of automated testing (something to jiggle around whitespace in all of the behavior tests and check they work the same), and I don't see any particularly compelling reason to treat the ZON behavior tests differently to the other Zig behavior tests.

Reply:
I’m open to more thorough tests being added for this in the future, but in the meantime, normalizing test data strictly decreases the quality of the tests and provides no upsides

Reply:
I do see your point -- I'll defer to @andrewrk's opinion on this.